app/views/dams_objects/show.html.erb

```diff
@@ -47,6 +47,7 @@
 		      <li><%= link_to "<i class='icon-plus-sign'></i> RDF View".html_safe, rdf_dams_object_path(@document['id']), class: "btn btn-block btn-small btn-link"%>
 		      <li><%= link_to "<i class='icon-plus-sign'></i> Data View".html_safe, data_dams_object_path(@document['id']), class: "btn btn-block btn-small btn-link"%>
 		      <li><%= link_to "<i class='icon-plus-sign'></i> DAMS 4.2 Preview".html_safe, dams42_dams_object_path(@document['id']), class: "btn btn-block btn-small btn-link"%>
+              <li><%= link_to "<i class='icon-plus-sign'></i> Mint DOI".html_safe, ezid_dams_object_path(@document['id']), class: "btn btn-block btn-small btn-link"%>

```

Comment:
I thought this was geared towards minting DOI's at the Collection level at this point? Should this be here instead, or at least in addition to, the Object view?

https://github.com/ucsdlib/damspas/blob/develop/app/views/dams_collections/_show_editor_tools.html.erb